consensus/quorum/quorum.go

```diff
@@ -201,20 +202,18 @@ func (s *cIdentities) IndexOf(pubKey bls.SerializedPublicKey) int {
 }
 
 // NthNext return the Nth next pubkey, next can be negative number
-func (s *cIdentities) NthNext(pubKey *bls.PublicKeyWrapper, next int) (bool, *bls.PublicKeyWrapper) {
-	found := false
-
+func (s *cIdentities) NthNext(pubKey *bls.PublicKeyWrapper, next int) (*bls.PublicKeyWrapper, error) {
```

Comment:
NthNext seems to be an unused function ?

Reply:
ok my bad i just saw you use for view change, do we have to use NthNext for view change and NthNextValidator seperately ? can we combine both ?

Reply:
no, doesn't look like we can use both right now